robot.html

```diff
@@ -1,6 +1,7 @@
 <html>
 <head>
     <script type="text/javascript" src="http://libs.baidu.com/jquery/2.1.1/jquery.min.js"></script>
+    <script type="text/javascript" src="connection.js"></script>
```

Comment:
this file is not uploaded, each user need to create it by himself. if the user don't have corresponding info than he is not able to use the system.